### PR TITLE
Build python wheels for aarch64

### PR DESCRIPTION
**Pull request by edgarriba**

### CODE REVIEW

**Review by ianBtesser — COMMENTED** [review]
Cool

File: .github/workflows/pypi-release.yml [hunk 1]
```diff
@@ -17,23 +17,24 @@ jobs:
...
-      #  with:
-      #    platforms: all
+      - name: Set up QEMU
+        if: runner.os == 'Linux'
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: all
       - name: Install dependencies
-        run: python3 -m pip install --upgrade pybind11 setuptools wheel cibuildwheel==2.16.2
+        run: python3 -m pip install --upgrade pybind11 setuptools wheel cibuildwheel==2.3.1
```
Comment (mihelich) [inline .github/workflows/pypi-release.yml:26]: Is the constraint `cibuildwheel==2.3.1` just for build consistency, or is there a specific reason? Worth adding a comment.

**Review by mihelich — APPROVED** [review]
Thanks for solving this!

File: .github/workflows/pypi-release.yml (hunk 1 above)
Comment (edgarriba) [reply .github/workflows/pypi-release.yml:26]: that's a mistake actually :)

**Pull request merged**